import java.rmi.server.ExportException;

public class Exception {

   /* public static void main(String[] args) {
        int a = 5;
        int c = a / 0;                  // when we divide any no. with zero, it throws arithmetic exception
                                        // it can be handled using try catch method

        System.out.println(c);

    }*/

    /*public static void main(String[] args) {
        int a[] ={ 2,4,5,6};                                        //array index of bound exception

        System.out.println(a[9]);
    }*/

    // to handle we can use try-catch mechanism to handle the exceptions
    // try- multiple catch can ne used:  try- catch-catch-catch-catch
    // different catch for different kind of exceptions
}
